src/newmatrix.cpp

```diff
@@ -476,13 +475,12 @@ int system_test(rnum_t host, struct char_data *ch, int type, int software, int m
       }
     }
   }
-  snprintf(rollbuf, sizeof(rollbuf), "System test against %s with software %s: Starting TN %d", acifs_strings[type], programs[software].name, target);
 
   int skill = get_skill(ch, SKILL_COMPUTER, target) + MIN(GET_MAX_HACKING(ch), GET_REM_HACKING(ch));
   GET_REM_HACKING(ch) -= skill - get_skill(ch, SKILL_COMPUTER, detect);
   detect = 0;
 
-  snprintf(ENDOF(rollbuf), sizeof(rollbuf) - strlen(rollbuf), ", after get_skill %d, plus called modifier %d is %d", target, modifier, target + modifier);
```

Comment:
Where is this modifier coming from?

Reply:
It looks like it's used in `do_restrict` (which doesn't apply against IC, so appears to be NERP outside of decker PVP). I removed it here because it also gets added to rollbuf some 3 lines later under `if (modifier) ...`